lib/utils/constraints-utils.ts

```diff
@@ -0,0 +1,88 @@
+import { z } from "zod";
+
```

Comment:
jsdocs (/***/) comments around each exported member. Applies to each exported member on this file: type, function, interface.

Reply:
Oh interfaces also